imagine-dragons-web/contact.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+
+<html>
+
+  <head>
+    <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
+    <title>The website of Imagine Dragons</title>
+  </head>
+
+  <body>
+    <div id="nav-placeholder"></div>
+
+    <main>
+        <h1>Contact Imagine Dragons!</h1>
+        
+        <form>
+        
+        <label for="firstName">First Name:</label>
+	    <input type="text" class="text" id="firstName" required/>
+
+        <label for="lastName">Last Name:</label>
+        <input type="text" class="text" id="lastName" required/>
+        
+        <label for="birthDate">Birth Date:</label>
```

Comment:
be consistent with your naming. you have `id="footer-placeholder"` and you have `id="birthDateYears"`